proxy/pom.xml

```diff
@@ -49,6 +49,11 @@
       <artifactId>jaeger-thrift</artifactId>
       <version>0.31.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.skywalking</groupId>
+      <artifactId>receiver-proto</artifactId>
+      <version>7.0.0</version>
+    </dependency>
```

Comment:
Can we include the jaeger-proto and grpc artifacts directly or does this library provide something on top that we need? Thinking that this library might add bloat and make it harder to maintain and update dependencies

Reply:
Yeah I checked the `jaeger-proto` maven pkg, for some reason it doesn't include this https://github.com/jaegertracing/jaeger-idl/blob/master/proto/api_v2/collector.proto, so I have to switch to that one.

Reply:
switched to opentelementry one